tutorials/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tutorials
+suppressRelated: true
+---
+
+These tutorials will give you step-by-step instruction to help you learn NServiceBus.
+
+### [NServiceBus 101: Messaging Basics](nservicebus-101/)
+
+In this course, you will learn how to get working quickly with NServiceBus, including how to set up your development environment and create messaging endpoints, defining messages and handlers, how to send messages and publish events, and how to deal with exceptions and take advantage of automatic retries.
+
+### Additional courses
+
+Please [sign up for updates](https://particular.net/s/new-course-signup) if you'd like to be notified when we release additional courses.
```

Comment:
@SimonCropp I don't agree on that, I think that "It's a specific metric we can use to gauge interest in more courses and then prioritize their creation accordingly." makes sense in this context. We can set calendar reminder to revisit this in 2-3 months and decide if that should be removed.